TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/wrist/WristCommands.java

```diff
@@ -0,0 +1,22 @@
+package org.firstinspires.ftc.teamcode.subsystems.wrist;
+
+import com.arcrobotics.ftclib.command.FunctionalCommand;
+import com.arcrobotics.ftclib.command.Subsystem;
+
+import org.firstinspires.ftc.teamcode.R;
+import org.firstinspires.ftc.teamcode.Robot;
+
+public class WristCommands {
+
+    public static FunctionalCommand moveToStateCommand(WristState state){
+
+        return new FunctionalCommand(
+                () -> Robot.getInstance().WRIST.setState(state),
+                () -> {},
+                interrupt -> Robot.getInstance().WRIST.stop(),
```

Comment:
do you really want to stop at the end? why not keep aiming to target all the time?